bedrock/firefox/templates/firefox/australis/growth-firstrun-test1.html

```diff
@@ -0,0 +1,149 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/australis/tour-base.html" %}
+
+{% set_lang_files "firefox/australis/firefox_tour" %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{ _('Welcome to Firefox') }}{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+
+{% block body_id %}firstrun{% endblock %}
+
+{% block telemetry_id %}australis-tour-firstrun-35-test1{% endblock %}
+
+{% block doorhanger_text %}
+  data-text="{{ _('To get more from Firefox, take a minute to complete this one-time set-up.') }}"
+{% endblock %}
+
+{% block doorhanger_title %}
+  data-title="{{ _('Get set up now') }}"
+{% endblock %}
+
+{% block site_css %}
+  {{ css('growth_firstrun_test1') }}
+{% endblock %}
+
+{% block js %}
+<script>
```

Comment:
I think a global or name-spaced var is fine :+1:. But we should probably try and avoid the inline script still?